crates/egui_plot/src/lib.rs

```diff
@@ -768,6 +768,10 @@ impl Plot {
 
         // Determine position of widget.
         let pos = ui.available_rect_before_wrap().min;
+        // Minimum values for screen protection
+        min_size.x = min_size.x.at_least(64.0);
+        min_size.y = min_size.y.at_least(15.0);
```

Comment:
I don't think we should do this. If the user wants a plot that is `16x32` pixels big, let them have it

Reply:
> I don't think we should do this. If the user wants a plot that is `16x32` pixels big, let them have it

If it is greater than `1.0`, panic does not occur, so it was set to `1.0`.
If you want to edit it to a different size, please do so.